chess/5-pregame/pregame.md

```diff
@@ -151,13 +151,19 @@ For this phase the TAs will grade the quality of your project's source code. The
 
 All of the tests in your project must succeed in order to complete this phase.
 
-To pass off this assignment submit your work to the course [auto-grading](https://cs240.click/) tool. When that is done, meet with a TA and demonstrate that your Chess client and server meet all requirements and assign you a final grade.
+To pass off this assignment submit your work to the course [auto-grading](https://cs240.click/) tool. When that is done, meet with a TA and demonstrate that your Chess client and server meet all requirements and they will assign you a final grade.
```

Comment:
This seems like it is saying the same thing twice. Just with an exclamation point the second time.

Could it just say:

To pass off this assignment submit your work to the course [auto-grading](https://cs240.click/) tool. Once it completely passes the auto-grader, meet with a TA and demonstrate that your Chess client and server meet all requirements and they will assign you a final grade.